astropy/visualization/wcsaxes/core.py

```diff
@@ -307,25 +332,179 @@ def plot_coord(self, *args, **kwargs):
 
             plot_data = []
             for coord in self.coords:
-                if coord.coord_type == 'longitude':
+                if coord.coord_type == "longitude":
                     plot_data.append(frame0.spherical.lon.to_value(u.deg))
-                elif coord.coord_type == 'latitude':
+                elif coord.coord_type == "latitude":
                     plot_data.append(frame0.spherical.lat.to_value(u.deg))
                 else:
-                    raise NotImplementedError("Coordinates cannot be plotted with this "
-                                              "method because the WCS does not represent longitude/latitude.")
+                    raise NotImplementedError(
+                        "Coordinates cannot be plotted with this "
+                        "method because the WCS does not represent longitude/latitude."
+                    )
 
-            if 'transform' in kwargs.keys():
-                raise TypeError("The 'transform' keyword argument is not allowed,"
-                                " as it is automatically determined by the input coordinate frame.")
+            if "transform" in kwargs.keys():
+                raise TypeError(
+                    "The 'transform' keyword argument is not allowed,"
+                    " as it is automatically determined by the input coordinate frame."
+                )
 
             transform = self.get_transform(native_frame)
-            kwargs.update({'transform': transform})
+            kwargs.update({"transform": transform})
 
             args = tuple(plot_data) + args[1:]
 
         return super().plot(*args, **kwargs)
 
+    def add_beam(
+        self,
+        header=None,
+        major=None,
+        minor=None,
+        angle=None,
+        corner="bottom left",
+        frame=False,
+        borderpad=0.4,
+        pad=0.5,
+        **kwargs,
+    ):
+        """
+        Display the beam shape and size
+
+        By default, this method will search for the ``BMAJ``, ``BMIN``, and ``BPA``
+        keywords in the FITS header to set the major and minor axes and the
+        position angle on the sky.
```

Comment:
I don't think this should be mentioned here - rather, below in the ``header`` kwarg description, mention that if specified those keywords will be searched.

Reply:
Done in 3f903d5a1.